pkg/rbac/cache.go

```diff
@@ -34,5 +37,8 @@ var cacheInst = Cache{
 	usersLock:        sync.Mutex{},
 	shared:           SharedData{},
 	users:            map[string]*userData{},
+	restConfig:       config.GetClientConfig(),
 	pool:             db.GetConnection(),
```

Comment:
what happens if pool is nil here? Will we try again before sending queries?

Reply:
Good point. We can add a retry.

Reply:
Tested this with @Anxhela21 - If postgres is down, when api comes up, the UI will never recover, since we will keep erroring out trying to get clusterscoped resources. Error is `nil pointer dereference`